packages/components/src/internal/components/settings/AuditSettings.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+
+import { PermissionTypes } from '@labkey/api';
+
+import { RequiresPermission } from '../base/Permissions';
+import { LabelHelpTip } from '../base/LabelHelpTip';
+import {
+    getAppHomeFolderPath,
+    isAssayEnabled,
+    isDataChangeCommentRequirementFeatureEnabled,
+    isELNEnabled,
+    isWorkflowEnabled,
+} from '../../app/utils';
+import { useServerContext } from '../base/ServerContext';
+import { useAppContext } from '../../AppContext';
+
+export const AuditSettings: FC = () => {
+    const { api } = useAppContext();
+    const { container, moduleContext } = useServerContext();
+    const [isRequired, setIsRequired] = useState<boolean>(false);
+
+    useEffect(() => {
+        (async () => {
+            const settings = await api.folder.getAuditSettings(getAppHomeFolderPath(container, moduleContext));
+            setIsRequired(settings.requireUserComments);
+        })();
+    }, []);
+
+    const onDisableRequirement = useCallback(() => {
+        setIsRequired(false);
+        api.folder.setAuditCommentsRequired(false, getAppHomeFolderPath(container, moduleContext));
+    }, [api]);
+
+    const onEnableRequired = useCallback(() => {
+        setIsRequired(true);
+        api.folder.setAuditCommentsRequired(true, getAppHomeFolderPath(container, moduleContext));
```

Comment:
minor: could put `getAppHomeFolderPath(container, moduleContext)` into a variable instead of calling it multiple times